### PR TITLE
docs: update theming and game configuration, add message screen api doc

### PR DESCRIPTION
**Pull request by ThmsKnz**
Documentation update

### CODE REVIEW

File: docs/api/message_screen.gd.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+**Extends:** [CanvasLayer](../CanvasLayer)
+
+## Description
+
+A message screen shown when when a game is loaded/saved
+
+## Method Descriptions
+
+### show\_message
```
Comment (dploeger) [inline docs/api/message_screen.gd.md:13]: Might need to add code comments to enable a description here

**Review by dploeger — CHANGES_REQUESTED** [review]
Just a small thing. Thanks!

**Comment by ThmsKnz** [comment]
Thanks for spotting this. I've updated the commit.
I'll create a pull request including the comments together with the debug message feature which I'm testing currently.

**Comment by ThmsKnz** [comment]
API docs and theming.md have been updated to include also new scene name display in debug mode.

**Pull request merged**